src/coreComponents/finiteElement/elementFormulations/H1_Tetrahedron_Lagrange1_Gauss1.hpp

```diff
@@ -238,20 +246,22 @@ class H1_Tetrahedron_Lagrange1_Gauss1 final : public FiniteElementBase
 
 /// @cond Doxygen_Suppress
 
+template< typename SUBREGION_TYPE >
 GEOSX_FORCE_INLINE
 void H1_Tetrahedron_Lagrange1_Gauss1::
   fillMeshData( NodeManager const & GEOSX_UNUSED_PARAM( nodeManager ),
                 EdgeManager const & GEOSX_UNUSED_PARAM( edgeManager ),
                 FaceManager const & GEOSX_UNUSED_PARAM( faceManager ),
-                CellElementSubRegion const & GEOSX_UNUSED_PARAM( cellSubRegion ),
-                MeshData & GEOSX_UNUSED_PARAM( initialization ) )
+                SUBREGION_TYPE const & GEOSX_UNUSED_PARAM( cellSubRegion ),
+                MeshData< SUBREGION_TYPE > & GEOSX_UNUSED_PARAM( initialization ) )
```

Comment:
@francoishamon @rrsettgast @joshua-white What was the final decision about using `GEOSX_UNUSED_PARAM`? I vaguely recall that at some point we were considering removing it from the function declaration, and used instead `GEOSX_UNUSED_VAR` in the function body.